cross/libass/PLIST

```diff
@@ -0,0 +1,3 @@
+lib:lib/libass.so
```

Comment:
This PLIST is inverted ;) This entry should be prefixed with `lnk:`, the entry on L3 should be prefixed with `lib:`